schema.sql

```diff
@@ -697,6 +697,27 @@ CREATE OR REPLACE ACTION create_credentials_by_dwg(
         error('credentials issuer must be an issuer of delegated write grant (issuer_auth_public_key = dwg_issuer_public_key)');
     }
 
+
+    -- Get the public key for XRPL/NEAR wallets from database
+    $dwg_owner_public_key TEXT := null;
+    for $wallet_row in SELECT public_key FROM wallets 
+            WHERE ((wallet_type = 'XRPL' AND address = $dwg_owner) 
+                OR (wallet_type = 'NEAR' AND address = $dwg_owner))
+            LIMIT 1 {
+        $dwg_owner_public_key := $wallet_row.public_key;
+        break;
+    }
+     -- First, determine the wallet type of dag_owner
+    $dwg_owner_wallet_type TEXT := null;
+    for $type_row in SELECT wallet_type FROM wallets WHERE address = $dwg_owner LIMIT 1 {
+        $dwg_owner_wallet_type := $type_row.wallet_type;
+        break;
+    }
+
+    if $dwg_owner_wallet_type = 'XRPL' AND $dwg_owner_public_key IS NULL {
+        error('dwg_owner public key not found');
+    }
+
```

Comment:
I don't understand why you do 3 DB requests when it can be done in 1, as i proposed [here](https://github.com/idos-network/idos-schema/pull/82/files#r2113649821)